R/FilterStateEmpty.R

```diff
@@ -77,10 +77,11 @@ EmptyFilterState <- R6::R6Class( # nolint
     #'
     get_call = function() {
       filter_call <- if (isTRUE(self$get_keep_na())) {
-        call("is.na", private$get_varname_prefixed())
+        sprintf("is.na(%s)", private$get_varname_prefixed())
```

Comment:
I don't think `sprintf` makes sense in this. We don't need to make a string just to convert it to the `lang` instantly. 

`"FALSE"` could be `as.name("FALSE")` then.

Reply:
This was done for consistency with other classes.

Note that `call("is.na", private$get_varname_prefixed())` is insufficient as `private$get_varname_prefixed()` returns character now, so using `call` would require `call("is.na", as.name(private$get_varname_prefixed()))`.